### DISCUSSION TITLE
📣 [10/05/23] New Mojo Release (0.4.0)

### DISCUSSION THREAD

**Discussion by ematejska** [modular/modular#999] [Announcements]
Hi everyone :wave:,

We just released an update to Mojo:fire:(v0.4.0) with new features including default parameter values and a file module for basic I/O support. The parallelize function signature has changed so you no longer need to provide a runtime, as the runtime is now global. There are many more changes and bug fixes, for full release notes please [check the changelog](https://docs.modular.com/mojo/changelog.html#v0.4.0-2023-10-05).
To update mojo make sure you’re on the latest `modular` cli and then reinstall:
```
sudo apt-get update
sudo apt-get install modular
modular clean
modular install mojo
```
If you don’t have the SDK yet get started with Mojo at https://www.modular.com/mojo
Don’t forget to download our [VS code extension](https://marketplace.visualstudio.com/items?itemName=modular-mojotools.vscode-mojo), and check out our [Mojo examples in Github](https://github.com/modularml/mojo/tree/main/examples)!

Cheers! :beers:

**Comment by drunkwcodes** [comment]
Well done, Modular!
